src/test/java/org/mule/extension/validation/AggregateValidationTestCase.java

```diff
@@ -92,13 +92,15 @@ public void twoFailuresInAny() throws Exception {
 
   @Test
   public void nonValidationErrorInsideAny() throws Exception {
-    expected.expectErrorType("MULE", EXPRESSION.getType());
-    configureGetAllRunner(flowRunner("anyWithNonValidationError"), VALID_EMAIL, VALID_URL).run();
+    FlowRunner runner = configureGetAllRunner(flowRunner("anyWithNonValidationError"), VALID_EMAIL, VALID_URL);
+
+    assertThat(runner.buildEvent().getMessage().getPayload().getValue(),
+               is(sameInstance(runner.run().getMessage().getPayload().getValue())));
   }
 
   @Test
   public void nonValidationErrorMixedWithValidationErrorsInsideAny() throws Exception {
-    expected.expectErrorType("MULE", EXPRESSION.getType());
+    expected.expectErrorType(VALIDATION_NAMESPACE, MULTIPLE_ERROR);
```

Comment:
We should review this with the architects (@elcoya / @nicomz ). Although, the documentation it says that the error that should be thrown is an ERROR:MULTIPLE, the behavior of the connector threw another error (and the tests were in accordance with the behavior of the connector instead of the documents).

So the question is: 
Do you think this is a fix for behavior to work as expected (and as documented)?
Or is it a major release because it breaks backward compatibility?